ansible/roles/php70/tasks/main.yml

```diff
@@ -19,6 +19,17 @@
   notify:
     - restart php7.0-fpm
 
+- name: install zip, soap, simple_xml, mbstring
```

Comment:
@jonasartack this should be done i a project specific config: there is an example: https://github.com/vagrant-php/ubuntu/tree/14.04/ansible/roles/php-application-drupal7, it should work with php5 too